portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,53 +36,6 @@
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
       
-      //HTML Form Steps
```

Comment:
I guess this whole `DataServlet.java` file can be removed now because it doesn't do anything and is not used.